src/app/pages/MediaAssetPage/index.test.jsx

```diff
@@ -16,15 +16,19 @@ import mostWatchedData from '#data/pidgin/mostWatched/index.json';
 import igboPageData from '#data/igbo/cpsAssets/afirika-23252735';
 import getInitialData from '#app/routes/cpsAsset/getInitialData';
 
-const toggleState = {
-  mediaPlayer: {
-    enabled: true,
-  },
-};
+jest.mock('#containers/ChartbeatAnalytics', () => {
```

Comment:
Should this be part of this PR? 

++ the other files adding the same:

src/app/pages/PhotoGalleryPage/index.test.jsx
src/app/pages/StoryPage/index.test.jsx

The associated snaps:

src/app/pages/MediaAssetPage/__snapshots__/index.test.jsx.snap
src/app/pages/PhotoGalleryPage/__snapshots__/index.test.jsx.snap

Reply:
Technically shouldn't but tests won't pass without it. Unfortunately ComScoreAnalytics is toggle-able, and when I removed the ToggleContextProvider the tests / snapshots failed. 

(Let me try removing now to see if it works)

Reply:
👍

Reply:
(it didn't work). I think previously these pages worked because there was a `toggleContext` within the MediaPlayer, but now that's no longer the case, the toggleContext is required at a higher level i.e. at the test MAP / STY / PGL Page setup